spec/factories/landing_pages.rb

```diff
@@ -1,11 +1,43 @@
 FactoryBot.define do
-  factory :landing_page, parent: :content_item do
+  factory :landing_page, class: LandingPage, parent: :content_item do
```

Comment:
Why does the class need to be specified now?

Reply:
(It should have been all along, we just got lucky that the previous tests we used it in didn't mind that they were actually getting a ContentItem)